### PR TITLE
Portfolio Mobile Version 

### PR DESCRIPTION
**Pull request by degisew**
This is the mobile version of my portfolio website which contains  my detail.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -113,17 +74,384 @@ section p {
...
+  background-color: inherit;
+  box-shadow: crimson;
+  width: 144px;
+  height: 48px;
+  border-radius: 10px;
+  font-weight: bolder;
+  font-size: 15px;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+}
```
Comment (1GORDON) [inline css/style.css:449]: Great job @degisew 👏

- [ ] Please implement the button interactions specified in the project requirements. The hover and enable effects as provided in the Figma design
- Figma button interactions
button interactions

File: html/index.html [hunk 2]
```diff
@@ -11,46 +11,209 @@
...
+                   <img src="../images/css.png" alt="" width="330">
+                 </li>
+                <li><img src="../images/Frameworks.png" alt="" width="350"></li>
+                <li><img src="../images/skills.png" alt="" width="350"></li>
+              </ul>
+
+            </div>
+      <!-- </div> -->
+ </div>
+    <!-- </div> -->
```
Comment (1GORDON) [inline html/index.html:217]: Great job @degisew 👏

- [ ] You may please remove all the commented code from your project as this is not a good practice.

File: css/style.css [hunk 3]
```diff
@@ -113,17 +74,384 @@ section p {
...
+  gap: 10px;
+  position: absolute;
+  width: 144px;
+  height: 14px;
+  left: 0;
+
+  /* top: calc(50% - 48px/2); */
+  margin: 15px auto 0 40px;
+
+  /* border: 1px solid #6070FF; */
```
Comment (1GORDON) [inline css/style.css:384]: Great work @degisew 👏

- [ ] You may please remove all the commented code from your style sheet.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @degisew. ✋

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages  ✔️ 
- Professional Readme ✔️
- Linters are passing ✔️
- Usage of Semantic tags ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by degisew** [comment]
@1GORDON  I don't understand the required changes and now I can't finish it because there is no power here. It is night time

**Review by codecaiine — APPROVED** [review]
Hi @degisew , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Good use of gitflow
:heavy_check_mark: Comments are meaningful
:heavy_check_mark: Great PR description
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html/index.html [hunk 4]
```diff
@@ -1,56 +1,185 @@
...
+      <input type="submit" value="Get my resume" class="get-my-resume-btn">
+      <ul class="skills">
+        <li><img src="../images/languages.png" alt="languages"></li>
+        <li><img src="../images/js.png" alt=""  class="lang-list"></li>
+        <li><img src="../images/html.png" alt="" class="lang-list"></li>
+        <li><img src="../images/css.png" alt="" class="lang-list"></li>
+        <li><img src="../images/Frameworks.png" alt="frameworks"></li>
+        <li><img src="../images/skills.png" alt="skills" ></li>
+      </ul>
+    </section>
```
Comment (AmitKlohan) [inline html/index.html:181]: Hi @degisew 
You've done great work so far

Suggestion 

I suggest to add a footer in last section to make this more semantic

**Review by AmitKlohan — COMMENTED** [review]
Hi @digisew 

you may consider my suggestions in the comment

File: html/index.html [hunk 5]
```diff
@@ -1,56 +1,185 @@
...
-        <li><img src="../images/Linkedinicon.png" alt="" /></li>
-        <li><img src="../images/github.png" alt="" /></li>
-        <li><img src="../images/twitter.png" alt="" width="20" height="20" /></li>
-        <li><img src="../images/M.png" alt="" width="20" height="20" /></li>
-        <li><img src="../images/fingers.png" alt="" width="20" height="20" /></li>
-      </ul>
-      
+      <section class="sub-section-of-1">
+      <div class="primary-text">
+        <p>
```
Comment (teke85) [inline html/index.html:20]: @degisew Great job so far!! I suggest you use an H1 tag as opposed to using a p tag as per the HTML Semantics tags, and HTML and CSS best practices.

**Review by scientistsagoemarcus — COMMENTED** [review]
Good job so far, you can still work harder.

**Pull request merged**